src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -230,7 +230,8 @@ export const validateAddress = (
       }));
     const confirmedSuggestions = suggestedAddresses.filter(
       suggestion =>
-        suggestion.addressMetaData?.deliveryPointValidation === CONFIRMED,
+        suggestion.addressMetaData?.deliveryPointValidation === CONFIRMED ||
+        suggestion.addressMetaData?.addressType === 'International',
```

Comment:
What do you think of if we did `suggestion.addressMetaData?.addressType.toLowerCase() === international`? Just in case the casing every changes?